### PR TITLE
dev ontoubemmp refactor saref oema concepts

### PR DESCRIPTION
**Pull request by qhouyee**
This pull request has updated ontoubemmp to be linked to OntoBIM and OntoDevice as a domain ontology for building utilities and energy. There are also corrections to some concepts and relationships as well as the removal of external namespaces

### CODE REVIEW

File: JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv [hunk 1]
```diff
@@ -3,45 +3,37 @@ OntoUBEMMP,TBox,https://www.theworldavatar.com/kg/ontoubemmp/,https://www.w3.org
...
-hasArea,Object Property,,,http://www.purl.org/oema/enaeq/Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-SolarCollectors,Class,SolarEnergyGenerator,IS-A,,,,Solar collector: converts solar radiation into usable thermal energy,https://www.theworldavatar.com/kg/ontoubemmp/,
+consumesEnergy,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device UNION https://w3id.org/bot#Building ,EnergyConsumption,,A relationship between a facility and its consumed energy,https://www.theworldavatar.com/kg/ontoubemmp/,
+producesEnergy,Object Property,,,SolarDevice,EnergySupply,,A relationship that links the solar device to the energy it produces,https://www.theworldavatar.com/kg/ontoubemmp/,
+hasArea,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,,https://www.theworldavatar.com/kg/ontoubemmp/,
+SolarCollectors,Class,SolarDevice,IS-A,,,,Solar collector: converts solar radiation into usable thermal energy,https://www.theworldavatar.com/kg/ontoubemmp/,
```
Comment (sm453) [inline JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv:28]: Wouldn't it be less confusing to use singular `SolarCollector`, i.e. removing the 's' from this and related classes? The comment uses singular...

File: JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv [hunk 2]
```diff
@@ -83,17 +53,7 @@ receivesRadiation,Object Property,,,https://w3id.org/bot#Building UNION SolarDev
...
-WallPVTTubeCollectors,Class,PVTTubeCollectors,IS-A,,,,PVT Tube Collectors with wall type,https://www.theworldavatar.com/kg/ontoubemmp/,
-EastWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with east orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-WestWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with west orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-NorthWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with north orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
-SouthWallPVTTubeCollectors,Class,WallPVTTubeCollectors,IS-A,,,,Wall PVT Tube Collectors with south orientation,https://www.theworldavatar.com/kg/ontoubemmp/,
+hasSolarSuitableArea,Object Property,,,https://www.theworldavatar.com/kg/ontobuildingstructure/RoofFacade UNION https://www.theworldavatar.com/kg/ontobuildingstructure/WallFacade,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,A relationship linking the building's roof or wall fa�ade with their area that is suitable for receiving solar energy.,https://www.theworldavatar.com/kg/ontoubemmp/,
```
Comment (sm453) [inline JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv:58]: Please fix the dodgy character in this and the subsequent line (and regenerate the owl). You'll need to find the right UTF-8 'ç'.

File: JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv (hunk 1 above)
Comment (sm453) [reply JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv:22]: Are you sure you don't want to change this?

Comment (qhouyee) [reply JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv]: Sorry, my mistake. I missed this out previously. Thank you for catching it :)

File: JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv (hunk 2 above)
Comment (qhouyee) [reply JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv:58]: I swapped it to the normal c, if that is fine.

**Review by sm453 — CHANGES_REQUESTED** [review]

**Review by sm453 — APPROVED** [review]

**Pull request merged**